homeassistant/components/tasmota/fan.py

```diff
@@ -56,42 +59,45 @@ def __init__(self, **kwds):
         )
 
     @property
-    def speed(self):
-        """Return the current speed."""
-        return TASMOTA_TO_HA_SPEED_MAP.get(self._state)
+    def speed_count(self) -> Optional[int]:
```

Comment:
This will always return an integer.

Reply:
Fixed by https://github.com/home-assistant/core/pull/47175